client/source/utilities/axiosRequests.js

```diff
@@ -1,18 +1,52 @@
 import axios from 'axios';
 
 
-const authServerURL = 'http://localhost:3000/api';
-
+const authServerURL = 'http://localhost:3000';
 
+/**
+ * Fire an Axios Get Request.
+ *
+ * It takes a string value to append to the endpoint,
+ * which should match the Atelier API endpoint pattern.
+ * @param {string} endpoint - Endpoint to be used
+ * @returns an Axios Request Promise
+ * */
 const get = function getByEndpoint(endpoint) {
-  return axios.get(`${authServerURL}/${endpoint}`);
+  return axios.get(`${authServerURL}/api/${endpoint}`);
```

Comment:
A suggestion to leave the slash after api out from the axiosRequests.js file? I'm thinking about when any one of us makes a request on our module, I'd like to use /endpoint vs. just endpoint, as I use the variable "reviews", which also happens to be the endpoint I'm using. This way, the /reviews can help distinguish.

Reply:
Easy fix, sure!